README.md

```diff
@@ -12,7 +12,7 @@ Currently Font Awesome Free, version 6.7.2 is included.
 ## Requirements
 
 - PHP 7.4.3 or later
```

Comment:
Since we're bumping a major version, we might as well drop PHP 7 support too.

Reply:
There wasn't a need to bump the PHP version higher as MW 1.39 supports PHP 7.4.3

Reply:
Yes, but that PHP is EOL for years and forces us to stick with old code. 

Modernising the code to PHP 8 (and dropping 7) later will require another major version bump. My thinking is if we're doing a major version anyway (because of dropping MW support), then it might be a good time to drop other EOL dependencies. This allows us to do cleanup later as part of the same major release.

On the other hand, we did not drop support in Maps, and also see https://github.com/ProfessionalWiki/Maps/pull/793.

There's not a lot of code here, and very little active development, so I guess there's no urgent need to drop it.

Reply:
>  This allows us to do cleanup later as part of the same major release.

Did you mean "minor" release?

`major.minor.patch`

For Composer-based extensions/packages, there is no need to maker a major release when raising minimum PHP version, because Composer already holds back new versions that don't work on your system. It's different for raising MW requirements, since those are not tracked by Composer. See also the SMW release policy: https://github.com/SemanticMediaWiki/SemanticMediaWiki/blob/master/docs/RELEASE-POLICY.md